src/Symfony/Component/Form/Extension/Validator/Type/FormTypeValidatorExtension.php

```diff
@@ -29,10 +29,23 @@ class FormTypeValidatorExtension extends BaseValidatorExtension
 {
     private ValidatorInterface $validator;
     private ViolationMapper $violationMapper;
-    private bool $legacyErrorMessages;
 
-    public function __construct(ValidatorInterface $validator, bool $legacyErrorMessages = true, FormRendererInterface $formRenderer = null, TranslatorInterface $translator = null)
+    /**
+     * @param ValidatorInterface         $validator
+     * @param FormRendererInterface|null $formRenderer
+     * @param TranslatorInterface|null   $translator
+     */
+    public function __construct(ValidatorInterface $validator, /*FormRendererInterface $formRenderer = null, TranslatorInterface $translator = null*/)
```

Comment:
I'm not sure of my migration path. 

Is it the good way for deprecate/remove arg which is not at end of method ?